opentech/settings/base.py

```diff
@@ -674,3 +674,38 @@
 PROJECTS_ENABLED = False
 if env.get('PROJECTS_ENABLED', 'false').lower().strip() == 'true':
     PROJECTS_ENABLED = True
+
+
+# Debug Toolbar
+# https://django-debug-toolbar.readthedocs.io/en/latest/index.html
+if DEBUG:
```

Comment:
Isn't `DEBUG` always `False` at this point because of the definition at the top of the file, or am i missing something? Where as in `settings/dev.py` it is always `True`

Wouldn't it make sense to keep it int he dev settings and then there is no need to the debug check at all?

Also worth noting the vagrant box sets the default settings like so: 
https://github.com/OpenTechFund/opentech.fund/blob/3d2e02a16e38894282ebf2f5f79eab405a61c784/vagrant/provision.sh#L49-L52
Which is the preferred entrypoint for dev.

Reply:
🤦‍♂️

Too used to single settings files!